.env.example

```diff
@@ -1 +0,0 @@
-API_KEY= # add api key here
```

Comment:
You don't need to delete this - it's useful for anyone else who needs to set up the repo in future